pkg/controller/grafana/grafana_reconcile.go

```diff
@@ -485,3 +492,49 @@ func doCheckApplicationMonitoring(r *ReconcileGrafana) (bool, error) {
 	return false, nil
 
 }
+
+func cleanupCSMonitoring(r *ReconcileGrafana, cr *v1alpha1.Grafana) error {
+
+	collectdDep := utils.CollectdDeployment(cr)
```

Comment:
please add log entry indicating that cleanup was started

Reply:
@pgodowski , addressed it in the latest commit